linera-views/src/backends/scylla_db.rs

```diff
@@ -27,11 +27,11 @@ const MAX_BATCH_TOTAL_SIZE: usize = 16773216;
 /// Therefore the actual MAX_VALUE_SIZE might be lower.
 /// At the maximum the key_size is 1024 bytes (see below) and we pack just one entry.
 /// So if the key has 1024 bytes this gets us the inequality
-/// 1 + 1 + serialized_size(MAX_KEY_SIZE)? + serialized_size(x)? <= RAW_MAX_VALUE_SIZE
+/// 1 + 1 + 1 + serialized_size(MAX_KEY_SIZE)? + serialized_size(x)? <= RAW_MAX_VALUE_SIZE
 /// and so this simplifies to 1 + 1 + (2 + 10240) + (4 + x) <= RAW_MAX_VALUE_SIZE
 /// (we write 4 because get_uleb128_size(RAW_MAX_VALUE_SIZE) = 4)
 /// and so to a maximal value of 408569;
-const VISIBLE_MAX_VALUE_SIZE: usize = 16754728;
+const VISIBLE_MAX_VALUE_SIZE: usize = 16752727;
```

Comment:
Can we write this as:

```suggestion
const VISIBLE_MAX_VALUE_SIZE: usize =
    RAW_MAX_VALUE_SIZE - MAX_KEY_SIZE
        - get_uleb128_size(RAW_MAX_VALUE_SIZE)
        - get_uleb128_size(MAX_KEY_SIZE)
        - 3;
```

(Maybe we can make `get_uleb128_size` `const`?)

Also, identifiers like `RAW_MAX_VALUE_SIZE`, and everything in the comment that is basically code, should be in backticks.

Reply:
No, we cannot do that (the computer is complaining). Otherwise put the bactsticks and additional comments.